### ISSUE TITLE
TON-pool.com Stratum URL is not supported!

### ISSUE THREAD

**Issue by megalolka** [develsoftware/GMinerRelease#667]
Hello!
ton-pool.com has a stratum address with "/stratum" at the end:
stratum+tcp://eu1.stratum.ton-pool.com:443/stratum

But Gminer is not support the "/stratum" directory in the end of server URL.
Please fix it. I believe it's a very small fix and can be implemented quickly.

**Comment by rluik** [comment]
I assume the same issue applies to toncoinpool.io (?) or do we need a different arguments structure for wss pool?

**Comment by japone777** [comment]
https://next.ton-pool.com i use it adress

**Comment by megalolka** [comment]
> https://next.ton-pool.com i use it adress

Yes, it works. But it's not a new and reliable stratum server. And this old server will shut down soon.

**Comment by IUaena0516** [comment]
2.83 can connect ton-pool  , but no benefit.
![image](https://user-images.githubusercontent.com/96397768/156570629-f57f1350-429d-4dea-bf95-3abd2760b437.png)
